onnx/numpy_helper.py

```diff
@@ -8,6 +8,7 @@
 
 import numpy as np
 
+import onnx.custom_element_types as custom_np_types
```

Comment:
Curious if there was a need to rename? If not:

```suggestion
from onnx import custom_element_types
```

Reply:
I let this syntax since I rename the module with an underscore.

Reply:
Having lines too long means I struggle with the linter.